skbio/stats/composition.py

```diff
@@ -751,33 +766,37 @@ def ancom(table, grouping,
     >>> from skbio.stats.composition import ancom
     >>> import pandas as pd
 
-    Now let's load in a pd.DataFrame with 6 samples and 7 unknown bacteria:
+    Now let's load in a DataFrame with 6 samples and 7 features (e.g.,
+    these may be bacterial OTUs):
 
     >>> table = pd.DataFrame([[12, 11, 10, 10, 10, 10, 10],
     ...                       [9,  11, 12, 10, 10, 10, 10],
     ...                       [1,  11, 10, 11, 10, 5,  9],
     ...                       [22, 21, 9,  10, 10, 10, 10],
     ...                       [20, 22, 10, 10, 13, 10, 10],
     ...                       [23, 21, 14, 10, 10, 10, 10]],
-    ...                      index=['s1','s2','s3','s4','s5','s6'],
-    ...                      columns=['b1','b2','b3','b4','b5','b6','b7'])
+    ...                      index=['s1', 's2', 's3', 's4', 's5', 's6'],
+    ...                      columns=['b1', 'b2', 'b3', 'b4', 'b5', 'b6',
+    ...                               'b7'])
 
-    Then create a grouping vector.  In this scenario, there
-    are only two classes, and suppose these classes correspond to the
-    treatment due to a drug and a control.  The first three samples
-    are controls and the last three samples are treatments.
+    Then create a grouping vector. In this example, there is a treatment group
+    and a placebo group.
 
-    >>> grouping = pd.Series([0, 0, 0, 1, 1, 1],
-    ...                      index=['s1','s2','s3','s4','s5','s6'])
+    >>> grouping = pd.Series(['treatment', 'treatment', 'treatment',
+    ...                       'placebo', 'placebo', 'placebo'],
+    ...                      index=['s1', 's2', 's3', 's4', 's5', 's6'])
 
-    Now run ``ancom`` and see if there are any features that have any
-    significant differences between the treatment and the control.
+    Now run ``ancom`` to determine if there are any features that are
+    significantly different in abundance between the treatment and the placebo
+    groups. The first DataFrame that is returned contains the ANCOM test
+    results, and the second contains the percentile abundance data for each
+    feature in each group.
 
-    >>> results = ancom(table, grouping)
-    >>> results['W']
+    >>> ancom_df, percentile_df = ancom(table, grouping)
+    >>> ancom_df['W']
     b1    0
     b2    4
-    b3    1
+    b3    0
```

Comment:
The test became more conservative now that we're correcting for multiple comparisons by default.